app/src/app/navtitle/navtitle.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="navtitle-inner">
   <h1 i18n><a routerLink="/">Itero</a></h1>
-  <h2 i18n>Online iterative vote application</h2>
+  <h2 i18n>Online iterative vote <span class="appterme">application</span></h2>
```

Comment:
The class name is wrong. "terme" is [not an English word](https://www.lexico.com/search?filter=en_dictionary&query=terme). I propose "title-extension" or "long-title", but you can find something even better.